client/components/draft_input/components/input_caption.coffee

```diff
@@ -0,0 +1,212 @@
+React = require 'react'
+window.global = window
+window.process = {env: {NODE_ENV: 'development'}}
```

Comment:
Happens because it is running in the client, looks like sd can provide it instead